src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -475,12 +471,13 @@ private boolean isArrayParent(Node candidate) {
             return false;
         }
 
-        private Object childrenAsContentMapList(Node node) {
+        private List<ContentMap> childrenAsContentMapList(Node node) {
             try {
                 return asNodeStream(node.getNodes()).map(n -> new EntryableContentMap(config.cloneWith(n)))
                         .collect(Collectors.toList());
             } catch (RepositoryException e) {
-                throw new RuntimeException(e);
+                log.error("Failed to get children of node {}", node, e);
+                return new ArrayList<>();
```

Comment:
`Collections.EMPTY_LIST` rather than new object and really make those exceptions `log.debug()` only. The only way you can get exception here (if you dive in `o.a.j.core.NodeImpl.getNodes()` is either when node was deleted under your hands, in which case it is ok to act like it didn't exist in a first place or when someone removed access to the node from under your hands in which case it is again ok to act as if it doesn't exist. Whole JCR API is polluted by `RepositoryException` and while I'm not friend of sneaky throws, I also don't see a reason to transfer this pollution in the logs just for situations that are perfectly valid and out of control of the user.